wsimod/nodes/nodes.py

```diff
@@ -793,7 +793,25 @@ def __init__(self, capacity=0, area=1, datum=10, initial_storage=0):
         else:
             self.storage = self.empty_vqip()
             self.storage_ = self.empty_vqip()  # Lagged storage for mass balance
+    
+    def apply_overrides(self, overrides: Dict[str, Any] = {}):
+        """Apply overrides to the tank.
 
+        Enables a user to override any of the following parameters: 
+        area, capacity, datum.
+        
+        Args:
+            overrides (dict, optional): Dictionary of overrides. Defaults to {}.
+        """
+        self.capacity = overrides.pop("capacity", 
+                                      self.capacity)
```

Comment:
should we warn that capacity isn't recalculated?

```suggestion
                                      self.capacity)
        if "area" in overrides.keys():
            print("warning: when overriding area, capacity is not automatically recalculated")
```